gallery/src/pages/StylingComponents.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {GalleryTile} from '../gallery-components/GalleryTiles';
+
+const StylingRSCPage = () =>
+  <GalleryTile title="How to style your app when using RSC">
+    <p className="nx-p">
+      If you have styling needs that aren't covered by the styles included in React Shared Components (RSC),
+      then you're going to have to create your own styles.
+    </p>
+    <section className="nx-tile-subsection">
+      <header>
+        <h3 className="h3 nx-h3--tile-subsection">Page &amp; component specific stylesheets</h3>
+      </header>
+      <p className="nx-p">
+        Take a look at <code className="nx-code">/gallery/src/main.scss</code>. This file
+        contains custom styles for the Gallery and imports external stylesheets. Why have page and component
+        specific stylesheets? By grouping our styles together by use and (ideally) locating the SCSS
+        next to the HTML that it modifies we decrease clutter/noise in our main stylesheet, keep our styles focused
+        to what they modify, and also help maintain a clear separation between code, layout, and styling.
```

Comment:
This paragraph is pretty confusing.  The file that it asks the user to look at doesn't seem to be an example of either page-specific nor component-specific stylesheets.  In fact I don't think we even currently follow the "page-specific stylesheet" idea in the gallery. The reference to importing external stylesheets also comes across as a red-herring, as it might make one think that that is how component-specific styles get included when it isn't.

Reply:
> This paragraph is pretty confusing. The file that it asks the user to look at doesn't seem to be an example of either page-specific nor component-specific stylesheets. In fact I don't think we even currently follow the "page-specific stylesheet" idea in the gallery. The reference to importing external stylesheets also comes across as a red-herring, as it might make one think that that is how component-specific styles get included when it isn't.

Yeah, I just chopped the first couple of sentences: https://github.com/sonatype/sonatype-react-shared-components/pull/72/commits/2a802846baa276979c1ad29f2ecf7c4051d1d8af